core/src/test/java/org/infinispan/remoting/rpc/RpcManagerTest.java

```diff
@@ -207,7 +207,7 @@ public void testInvokeCommands() throws Exception {
       RpcManager rpcManager0 = cache(0).getAdvancedCache().getRpcManager();
 
       Exceptions.expectException(IllegalArgumentException.class, () -> {
-         rpcManager0.invokeCommands(Arrays.asList(address(0)), a -> command, MapResponseCollector.validOnly(),
+         rpcManager0.invokeCommands(Arrays.asList(address(1)), a -> command, MapResponseCollector.validOnly(),
```

Comment:
The topology id check is done in `RpcManagerImpl`, so I think this change isn't necessary.

Reply:
Hrmm it failed without this change.

Reply:
I see now, the validation is done by wrapping the command generator, so it actually runs inside `JGroupsTransport.invokeCommands`.